app/models/manageiq/providers/hawkular/middleware_manager/refresh_parser.rb

```diff
@@ -101,6 +110,7 @@ def alternate_machine_id(machine_id)
         # providers store it in downcase, so, we let the upcase/downcase logic to other methods with more
         # business knowledge.
         # 1 - https://bugzilla.redhat.com/show_bug.cgi?id=1294461
+        return if machine_id.length < 31
```

Comment:
@rubenvp8510 I don't see any change to how this method is being called so I'm not sure why this is in here, if there is an issue with this method can it be addressed in another PR?  As far as I can see the calling code is just moved but the argument is still just `eap.feed`